scripts/create_tables.sql

```diff
@@ -0,0 +1,259 @@
+	
+CREATE TABLE public."Skill"
+(
+    id_skill BIGSERIAL NOT NULL,
+    name VARCHAR,
+    PRIMARY KEY (id_skill)
+);
+
+ALTER TABLE IF EXISTS public."Skill"
+    OWNER to admin;	
+	
+	
+CREATE TABLE public."Entity_Skill"
+(
+    id_entity_skill BIGSERIAL NOT NULL,
+    id_entity BIGINT,
+    id_skill BIGINT,
+    PRIMARY KEY (id_entity_skill),
+    CONSTRAINT id_entity FOREIGN KEY (id_entity)
+        REFERENCES public."Entity" (id_entity) MATCH SIMPLE
+        ON UPDATE CASCADE
+        ON DELETE CASCADE,
+    CONSTRAINT id_skill FOREIGN KEY (id_skill)
+        REFERENCES public."Skill" (id_skill) MATCH SIMPLE
+        ON UPDATE CASCADE
+        ON DELETE CASCADE
+);
+
+ALTER TABLE IF EXISTS public."Entity_Skill"
+    OWNER to admin;
+	
+CREATE TABLE public."Equipment"
+(
+    id_equipment BIGSERIAL NOT NULL,
+    helmet BIGINT,
+    chest BIGINT,
+    leg_armor BIGINT,
+    boots BIGINT,
+    left_hand BIGINT,
+    right_hand BIGINT,
+    accessory BIGINT,
+    PRIMARY KEY (id_equipment),
+    CONSTRAINT helmet FOREIGN KEY (helmet)
+        REFERENCES public."Item" (id_item) MATCH SIMPLE
+        ON UPDATE SET NULL
+        ON DELETE SET NULL,
+    CONSTRAINT chest FOREIGN KEY (chest)
+        REFERENCES public."Item" (id_item) MATCH SIMPLE
+        ON UPDATE SET NULL
+        ON DELETE SET NULL,
+    CONSTRAINT leg_armor FOREIGN KEY (leg_armor)
+        REFERENCES public."Item" (id_item) MATCH SIMPLE
+        ON UPDATE SET NULL
+        ON DELETE SET NULL,
+    CONSTRAINT boots FOREIGN KEY (boots)
+        REFERENCES public."Item" (id_item) MATCH SIMPLE
+        ON UPDATE SET NULL
+        ON DELETE SET NULL,
+    CONSTRAINT left_hand FOREIGN KEY (left_hand)
+        REFERENCES public."Item" (id_item) MATCH SIMPLE
+        ON UPDATE SET NULL
+        ON DELETE SET NULL,
+    CONSTRAINT right_hand FOREIGN KEY (right_hand)
+        REFERENCES public."Item" (id_item) MATCH SIMPLE
+        ON UPDATE SET NULL
+        ON DELETE SET NULL,
+    CONSTRAINT accessory FOREIGN KEY (accessory)
+        REFERENCES public."Item" (id_item) MATCH SIMPLE
+        ON UPDATE SET NULL
+        ON DELETE SET NULL
+);
+
+ALTER TABLE IF EXISTS public."Equipment"
+    OWNER to admin;
+	
+	
+CREATE TABLE public."Player"
+(
+    id_player BIGSERIAL NOT NULL,
+    id_user BIGINT,
+    alignment VARCHAR,
+    backstory VARCHAR,
+    id_equipment BIGINT,
+    id_entity BIGINT,
+    PRIMARY KEY (id_player),
+    CONSTRAINT id_user FOREIGN KEY (id_user)
+        REFERENCES public."User" (id_user) MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION,
+    CONSTRAINT id_equipment FOREIGN KEY (id_equipment)
+        REFERENCES public."Equipment" (id_equipment) MATCH SIMPLE
+        ON UPDATE SET NULL
+        ON DELETE SET NULL,
+    CONSTRAINT id_entity FOREIGN KEY (id_entity)
+        REFERENCES public."Entity" (id_entity) MATCH SIMPLE
+        ON UPDATE CASCADE
+        ON DELETE CASCADE
+);
+
+ALTER TABLE IF EXISTS public."Player"
+    OWNER to admin;
```

Comment:
newline na końcu pliku
```suggestion
    OWNER to admin;
    
```